### DISCUSSION TITLE
Rust

### DISCUSSION THREAD

**Discussion by OmO-sketch** [Py-Contributors/support#380] [Q&A] [answered]
What is std::sync::Arc and how does it differ from Rc?

**Comment by Tvenus** [comment, marked as answer]
std::sync::Arc is an atomic reference-counted smart pointer for thread-safe shared ownership. Unlike Rc, Arc can be used across multiple threads because it uses atomic operations to manage reference counting, ensuring safety in concurrent contexts.
